File: 3-ano/exercicios-de-aula/murilo/ex01.php
<?php
    echo "<h1>Tabuada do 5</h1>";

    echo "<h2>Usando FOR</h2>";
    for ($i = 0; $i <= 10; $i++) {
        $resultado = 5 * $i;
        echo "5 * $i = $resultado <br>";
    }

    echo "<h2>Usando WHILE</h2>";
    $i = 0;
    while ($i <= 10) {
        $resultado = 5 * $i;
        echo "5 * $i = $resultado <br>";
        $i++;
    }

?>

File: 3-ano/exercicios-de-aula/murilo/aula-20-03/ex01.dart
void main() {
  String nome = "dart";
  int numero = 17;
  double valor = 5.5;
  bool checar = false;

  print("1: " + nome + " " + numero.toString() + " " + valor.toString() + " " + checar.toString());
  print("2: $nome $numero $valor $checar");

  for (int i = 0; i < 10; i++) {
    print(i);  
  }

  print("Tabuada do 5");
  for (int i = 1; i <= 10; i++) {
    int resultado = 5 * i;
    print("5 X $i = $resultado");
  }
}